contextual.go

```diff
@@ -108,6 +111,13 @@ func ContextualLogger(enabled bool) LoggerOption {
 	}
 }
 
+// FlushLogger provides a callback for flushing data buffered by the logger.
```

Comment:
Should this also be marked as experimental?

Reply:
Yes, I think so. Changed.

Reply:
good call